src/test/java/seedu/address/testutil/EateryBuilder.java

```diff
@@ -74,7 +89,10 @@ public EateryBuilder withCategory(String category) {
     }
 
     public Eatery build() {
-        return new Eatery(name, address, category, tags);
+        return new Eatery(name, isOpen, address, category, tags);
     }
 
+    public Eatery buildClose() {
+        return new Eatery(name, false, address, category, tags);
+    }
```

Comment:
why do you have a buildClose() when you can like just use .withIsOpen(false).build()?

Reply:
haha omy yea i created too many methods LOL

Reply:
does this mean u gna delete buildClose() HAHAHA i think it's more consistent if u do .withIsOpen()..